packages/core/src/data-editor/data-editor.tsx

```diff
@@ -3440,7 +3440,18 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
                 return gridRef.current?.damage(damageList);
             },
             getBounds: (col, row) => {
-                return gridRef.current?.getBounds(col + rowMarkerOffset, row);
+                if (col === undefined && row === undefined && scrollRef.current && canvasRef.current) {
```

Comment:
This wont pass linting, explicit booleans are required for checks, in this case `canvasRef.current !== undefined`

Reply:
Fixed it 👍